datafusion/core/src/execution/context.rs

```diff
@@ -1148,6 +1151,15 @@ pub struct SessionState {
     pub runtime_env: Arc<RuntimeEnv>,
 }
 
+impl Debug for SessionState {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("SessionState")
+            .field("session_id", &self.session_id)
+            // TODO should we print out more?
```

Comment:
There are a lot of other fields on SessionState -- I wasn't sure the value of printing them, so I left it as a future todo